docs/user-guide/plotting.ipynb

```diff
@@ -76,17 +79,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['Sample'].variances = np.random.rand(N)\n",
+    "d['Sample'].variances = np.square(np.random.rand(N))\n",
```

Comment:
Do we really need to square here? Was it because of the comment below, so the user can recognise the values that were actually inputed are easily reasable from the graph?

Reply:
I mainly put this since the error bars were almost invisibly small. Anotherway would have been to scale them.